build.sbt

```diff
@@ -1,27 +1,46 @@
 val scala2_12 = "2.12.15"
-val scala2_13 = "2.13.8"
+val scala2_13 = "2.13.12"
 
-val silencerVersion = "1.7.8"
+ThisBuild / scalaVersion       := scala2_13
+ThisBuild / majorVersion       := 9
+ThisBuild / isPublicArtefact   := true
```

Comment:
As the silencer plugin has been removed, the following code could be added to match the behaviour of `auth-client`.
`ThisBuild / scalacOptions    += "-Wconf:src=src_managed/.*:s" // silence all warnings on autogenerated files`

Reply:
👍 added in [this commit](https://github.com/hmrc/domain/pull/52/commits/84859f1ee5ab99b978f18e7c1e5f7e0e2e6c92b2)